planning/standards-status.md

```diff
@@ -0,0 +1,83 @@
+---
+description: This list collects standards that the Thunderbird family of applications currently at least partially implements or supports (in our code base, ignoring things like TCP which we inherit from upstream components). It reflects the state of the current latest main repo code.
+---
+
+## Mail
+
+| Spec Name                                                          | Desktop Status | Android Status | Standards Involvement      | Comment |
+|:------------------------------------------------------------------ |:--------------:|:--------------:|:-------------------------- |:------- |
+| [POP3](https://datatracker.ietf.org/doc/html/rfc1939)              |  &#10004;      |   &#10004;     |                            |         |
+| [IMAP 4r1](https://datatracker.ietf.org/doc/html/rfc3501)          |  &#10004;      |   &#10004;     |                            |         |
+| [IMAP IDLE](https://datatracker.ietf.org/doc/html/rfc2177)         |  &#10004;      |   &#10004;     |                            |         |
+| [SMTP](https://datatracker.ietf.org/doc/html/rfc821)               |  &#10004;      |   &#10004;     |                            |         |
+| [NNTP](https://datatracker.ietf.org/doc/html/rfc977)               |  &#10004;      |   &#10004;     |                            |         |
+
+## RSS Feeder
+
+| Spec Name                                                          | Desktop Status | Android Status | Standards Involvement      | Comment |
+|:------------------------------------------------------------------ |:--------------:|:--------------:|:-------------------------- |:------- |
+| [RSS](https://www.rssboard.org/rss-specification)                  |  &#10004;      |      N/A       |                            | Both [RSS 0.9](https://www.rssboard.org/rss-0-9-0) and [RSS 2.0](https://www.rssboard.org/rss-specification) are supported. |
+| [Atom](https://datatracker.ietf.org/doc/html/rfc4287)              |  &#10004;      |      N/A       |                            | Both [Atom 0.3](https://datatracker.ietf.org/doc/html/draft-ietf-atompub-format-00) and [Atom 1.0](https://datatracker.ietf.org/doc/html/rfc4287) are supported. |
```

Comment:
Should these each just be separate rows if they're separate protocols?

Reply:
Sure I can do that.